Collectors/DetailedCollector.py

```diff
@@ -291,7 +291,7 @@ def process_gstream_tpc(self, gstream, addr, sid):
              event["size"] = event.pop("Size")
 
              if event["source"].startswith('/store') or event["source"].startswith('/user/dteam'):
-                  wlcg_packet = wlcg_converter.ConvertGstream(event)
+                  wlcg_packet = wlcg_converter.ConvertGstreamTPC(event)
```

Comment:
Where else is the `ConvertGstream` function used?

Reply:
sure